### PR TITLE
added dark, grey, and bright classmethods

### PR DESCRIPTION
**Pull request by kvyh**
I added classmethods for default values for dark, grey, and bright time into `MoonIlluminationConstraint`

### CODE REVIEW

File: astroplan/constraints.py [hunk 1]
```diff
@@ -546,6 +546,48 @@ def __init__(self, min=None, max=None, ephemeris=None):
         self.max = max
         self.ephemeris = ephemeris
 
+    @classmethod
+    def dark(cls, **kwargs):
+        """
+        This will use default values (acceptable fractional illumination)
+        of max=0.25 with no minimum. You can input min= or max= to change
+        the values and ephemeris= if a different ephemeris is desired,
+        by default it uses ``astropy.coordinates.solar_system_ephemeris.set``
```
Comment (bsipocz) [inline astroplan/constraints.py]: This is nitpicking: Could you add double backticks around parameters (e.g ``min`` and ``max``) and remove the lone = from them. Also I suggest to make those links to ``astropy.coordinates.solar_system_ephemeris`` by removing the extra backticks, (and you can also add the ~ in the front)

Comment (bmorris3) [inline astroplan/constraints.py]: @kvyh While you're at it, it looks like some other docstrings have that double-ticked reference to `solar_system_ephemeris` (e.g. [see here](https://github.com/astropy/astroplan/blob/master/astroplan/constraints.py#L482))). Would you change those to `` `~astropy.coordinates.solar_system_ephemeris.set` `` also?

Comment (bsipocz) [inline astroplan/constraints.py]: Leave the ``.set`` ending out of it, there is no API link for that, and also it's irrelevant I think.

**Comment by kvyh** [comment]
Ok, I changed the way that it is written for increased clarity (hopefully)

**Comment by bmorris3** [comment]
Looks good, but needs some basic tests.

**Comment by kvyh** [comment]
Tests have been added.

**Comment by bmorris3** [comment]
Thanks @kvyh!

**Pull request merged**